cassandra/1.10/README.md

```diff
@@ -0,0 +1,369 @@
+  ],
+  "vip": "node.cassandra.l4lb.thisdcos.directory:9042"
+}
+```
+
+SSH into your DC/OS cluster to connect to your Cassandra cluster:
+
+```
+$ dcos node ssh --master-proxy --leader
+core@ip-10-0-6-55 ~ $
+```
+
+You are now inside your DC/OS cluster and can connect to the Cassandra cluster directly. Connect to the cluster using the `cqlsh` client:
+
+```bash
+core@ip-10-0-6-153 ~ $ docker run -ti cassandra:3.0.10 cqlsh --cqlversion="3.4.0" <HOST>
+```
+
+Replace `<HOST>` with an IP from the `address` field, which we retrieved by running `dcos cassandra connection`, above:
+
+```bash
+core@ip-10-0-6-153 ~ $ docker run -ti cassandra:3.0.10 cqlsh --cqlversion="3.4.0" 10.0.3.228
+cqlsh>
+```
+
+You are now connected to your Cassandra cluster. Let's create a sample keyspace called `demo`:
+
+```sql
+cqlsh> CREATE KEYSPACE demo WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 };
+```
+
+Next, create a sample table called `map` in the `demo` keyspace:
+
+```sql
+cqlsh> CREATE TABLE demo.map (key varchar, value varchar, PRIMARY KEY(key));
+```
+
+Insert some data in your table:
+
+```sql
+cqlsh> INSERT INTO demo.map(key, value) VALUES('Cassandra', 'Rocks!');
+cqlsh> INSERT INTO demo.map(key, value) VALUES('StaticInfrastructure', 'BeGone!');
+cqlsh> INSERT INTO demo.map(key, value) VALUES('Buzz', 'DC/OS is the new black!');
+```
+
+Query the data back to make sure it's persisted correctly:
+
+```sql
+cqlsh> SELECT * FROM demo.map;
+
+ key                  | value
+----------------------+-------------------------
+            Cassandra |                  Rocks!
+ StaticInfrastructure |                 BeGone!
+                 Buzz | DC/OS is the new black!
+
+(3 rows)
+```
+
+Next, delete some data:
+
+```sql
+cqlsh> DELETE FROM demo.map where key = 'StaticInfrastructure';
+```
+
+Query again to ensure that the row was deleted successfully:
+
+```sql
+cqlsh> SELECT * FROM demo.map;
+
+ key       | value
+-----------+-------------------------
+ Cassandra |                  Rocks!
+      Buzz | DC/OS is the new black!
+
+(2 rows)
+```
+
+## Uninstall
+
+To uninstall Cassandra:
+
+```bash
+$ dcos package uninstall cassandra
+WARNING: This action cannot be undone. This will uninstall [cassandra] and delete all of its persistent data (logs, configurations, database artifacts, everything).
+Please type the name of the service to confirm: cassandra
+Uninstalled package [cassandra] version [1.0.25-3.0.10]
+DC/OS Apache Cassandra service has been uninstalled.
+Please follow the instructions at https://docs.mesosphere.com/current/usage/service-guides/cassandra/uninstall to remove any persistent state if required.
+```
+
+Use the [framework cleaner](https://docs.mesosphere.com/1.10/deploying-services/uninstall/#framework-cleaner) script to remove your Cassandra instance from ZooKeeper and to destroy all data associated with it. The script requires several arguments, the values for which are derived from your service name:
```

Comment:
Might be worth noting that you need to do this from the master, just in case the reader doesn't read the full instructions via the link.